crates/bevy_core/src/name.rs

```diff
@@ -100,26 +100,31 @@ impl std::fmt::Debug for Name {
 ///     for (name, mut score) in &mut scores {
 ///         score.0 += 1.0;
 ///         if score.0.is_nan() {
-///             bevy_utils::tracing::error!("Score for {:?} is invalid", name);
+///             // use the Display impl to return either the Name
```

Comment:
This kind of implementation information seems a bit too in-depth for an example. Additionally, the part about debug being more verbose is imo implicitly to be expected.

Suggestion: move this information outside of the example (maybe a section `# Implementation`?) and remove references to `Debug`

Reply:
Thanks for this, have updated the code if you could check its what you meant.

Reply:
Perfect, thanks :)